modules/nw-metallb-configure-svc.adoc

```diff
@@ -0,0 +1,72 @@
+[id="nw-metallb-configure-svc_{context}"]
+= Configuring a service with MetalLB
+
+You can configure a load-balancing service to use an external IP address from an address pool.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Install the MetalLB Operator and start MetalLB.
+
+* Configure at least one address pool.
+
+* Configure your network to route traffic from the clients to the machine network for the cluster.
```

Comment:
Should 'host' network of node be used instead of machine network?

Reply:
Machine network and host network are unfortunately used interchangeably.  I prefer machine network because that is the term that is used in the [network configuration parameters](https://docs.openshift.com/container-platform/4.8/installing/installing_bare_metal/installing-bare-metal-network-customizations.html#installation-configuration-parameters-network_installing-bare-metal-network-customizations) section of the installation documentation.